sample/package.json

```diff
@@ -16,7 +16,7 @@
     "typescript": "3.7.5"
   },
   "dependencies": {
-    "@mattrglobal/jsonld-signatures-bbs": "0.7.0",
+    "@mattrglobal/jsonld-signatures-bbs": "file:..",
```

Comment:
I'll need to double check on this with @tplooker , but I think we were wanting to keep this versioned rather than moving with unstable. Otherwise this seems like a good way to keep them together.

Reply:
Yeap talked to @OR13 about this, either we shift to a model where the sample targets the head of master and CI will fail if the samples are out of date (e.g incorrect) or we keep the samples to using only pinned versions of the lib like it does now